frontend/src/libs/components/header/header.module.scss

```
@@ -0,0 +1,76 @@
+:root {
+  --logo-font-size: 24px;
+  --profile-data-font-size: 16px;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  width: 100%;
+  color: var(--white);
+  font-weight: 400;
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+  background: var(--gray-600);
+}
+
+.logoContainer {
+  display: flex;
+  align-items: center;
+  gap: 7px;
+  padding: 15px 40px;
+}
+
+.logoContainer > * {
+  cursor: pointer;
+}
+
+.headerLogo {
+  width: 32.14px;
+  height: 32.14px;
+  background-color: var(--yellow);
+  border-radius: 50%;
+}
+
+.headerLogoText {
+  font-size: var(--logo-font-size);
+  line-height: 103.5%;
+  cursor: pointer;
+}
+
+.headerSidebar {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 100%;
+  height: 100%;
+  padding-right: 55px;
+  padding-left: 95px;
+}
+
+.headerSectionTitle {
+  font-size: var(--profile-data-font-size);
+  line-height: 28px;
+  border-bottom: 2px solid #ffb61d;
+}
+
+.headerProfileData {
+  display: flex;
+  align-items: center;
+  gap: 7px;
+}
+
+.headerProfileData > * {
```

```suggestion
.headerProfileData {
```
avoid such selectors